src/bp/core/misc/code-sandbox.ts

```diff
@@ -91,3 +93,18 @@ export class SafeCodeSandbox {
     this.tmpDir.removeCallback()
   }
 }
+
+export class UntrustedSandbox {
+  @memoize(() => getCacheKeyInMinutes(5)) // cache the result of this function for 1 minute
+  static getSandboxProcessArgs() {
+    const exposedEnv = {
+      ..._.pickBy(process.env, (_value, name) => name.startsWith('EXPOSED_')),
+      ..._.pick(process.env, 'TZ', 'LANG', 'LC_ALL', 'LC_CTYPE')
```

Comment:
I'm not sure why you need to pass LC_ALL and LC_CTYPE if LANG is passed. I imagine you stumbled on a particular use case, what is it ?

Reply:
No I just thought I'd include all the ENV that really matter that are not confidential